athena-jdbc/src/main/java/com/amazonaws/connectors/athena/jdbc/manager/JdbcMetadataHandler.java

```diff
@@ -315,4 +324,17 @@ public abstract void getPartitions(
 
         return splitClauses;
     }
+
+    /**
+     * Converts an ARRAY column's TYPE_NAME provided by the jdbc metadata to an ArrowType.
+     * @param typeName The column's TYPE_NAME (e.g. _int4, _text, _float8, etc...)
+     * @param precision Used for BigDecimal ArrowType
+     * @param scale Used for BigDecimal ArrowType
+     * @return Utf8 ArrowType (VARCHAR)
+     */
+    protected ArrowType arrowTypeFromTypeName(String typeName, int precision, int scale)
+    {
+        // Default array support is List<Utf8>.
```

Comment:
this comment talks about arrays but this method name and signature has nothing to do with arrays.

Reply:
Changed function's name to getArrayArrowTypeFromTypeName and changed comment.